### PR TITLE
[pod-leader] Add pod leader strategy

### PR DESCRIPTION
**Pull request by UrsaMaritimus**
Changes proposed in this pull request:
- Add strategy that works with yield yak like farming on avalanche, like the pod leader

### CODE REVIEW

File: src/strategies/pod-leader/index.ts [hunk 1]
```diff
@@ -0,0 +1,518 @@
...
+
+  const userInfoChunks = arrayChunk(userInfoResponse, 1);
+
+  const tokenResponse = await multicall(network, provider, abi, tokenCalls, {
+    blockTag
+  });
+
+  for (let i = 0; i < userInfoChunks.length; i++) {
+    const value = userInfoChunks[i];
+    const score = await processValues(
```
Comment (ChaituVR) [inline src/strategies/pod-leader/index.ts:506]: Here we are calling `processValues` function for every address, which has multiple fetch calls (ex: getPrice), and it is not scalable in the case of a large number of voters, is there an alternative you can think of where you can get data from a single call?

 Here is the checklist to add any strategy:
https://github.com/snapshot-labs/snapshot-strategies#checklist-for-adding-a-new-strategy

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/pod-leader/index.ts (hunk 1 above)
Comment (UrsaMaritimus) [reply src/strategies/pod-leader/index.ts:336]: I have removed the getPrice to simplify it, just based off token weighting now. Should be able to scale properly now.

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
Strategy is live: https://snapshot.org/#/strategy/pod-leader

**Comment by ChaituVR** [comment]
Hi @UrsaMaritimus We noticed this strategy is causing memory issues on server and breaks everything, we will disable this soon, please feel free to contact us on https://discord.snapshot.org

**Pull request merged**